src/components/CreatePollModal/CreatePollForm.tsx

```diff
@@ -186,7 +186,7 @@ export const CreatePollForm = ({
   const handleChangeDuration = useCallback(
     (e: ChangeEvent<HTMLInputElement>) => {
       setIsDirty(true);
-      setDuration(e.target.value);
+      setDuration(e.target.value.replace(/\..*/, ''));
```

Comment:
Note that other languages might use other number separators (English -> `1.1`; German -> `1,1`). How about using a regex that explicitly defines which characters we want to allow?

`e.target.value.replace(/[^0-9]*/g,'');` -> "Replace all characters that are not between (0..9) with nothing"

Reply:
using this regex /[^0-9]*/g will accept the numbers after the dot for example 12.4 will be 124 and i tried to remove every number will be writing after the dot for example 12.4 will be 12

Reply:
Note that there is also the way of entering scientific notation to get a decimal number without a decimal point 😉  Like `1e-3`